tooling/minimize/src/bb.rs

```diff
@@ -777,6 +777,37 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
                 })
                 .collect();
```

Comment:
This value here is entirely ignore for tuple argument passing. That's confusing and also wasteful.

Please move this into the `else` branch  of `if fn_sig.abi() == rustc_abi::ExternAbi::RustCall` so that we only do one of the two `args` computations.